config/webpack/production.js

```diff
@@ -2,4 +2,7 @@ process.env.NODE_ENV = process.env.NODE_ENV || 'production'
 
 const environment = require('./environment')
 
+// https://github.com/rails/webpacker/issues/1235
+environment.config.optimization.minimizer[0].options.uglifyOptions.ecma = undefined; // the default, or 5 if you want it explicitely
```

Comment:
Je préfère qu'on soit epxlicite, et qu'on mette plutôt "ecma = 5; // for IE 11 support"